examples/Raylib/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.10)
+if (POLICY CMP0074)
+    cmake_policy(SET CMP0074 NEW)
+endif()
+
+project(LDtkLoader-Raylib)
+
+set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules ${CMAKE_MODULE_PATH})
+
+find_package(raylib REQUIRED)
+add_executable(LDtkRaylib main.cpp)
+target_include_directories(LDtkRaylib PRIVATE ../../include ${RAYLIB_INCLUDE_DIRS})
+target_link_libraries(LDtkRaylib PRIVATE ${RAYLIB_LIBRARIES} LDtkLoader)
```

Comment:
CMake variables are case sensitive, and according to `raylib-config.cmake` we have the following variables provided by raylib :
```
# Once done, this defines a raylib target that can be passed to
# target_link_libraries as well as following variables:
#
#   raylib_FOUND - System has raylib installed
#   raylib_INCLUDE_DIRS - The include directories for the raylib header(s)
#   raylib_LIBRARIES - The libraries needed to use raylib
#   raylib_LDFLAGS - The linker flags needed with raylib
#   raylib_DEFINITIONS - Compiler switches required for using raylib
````
So the correct names are with lowercase raylib :
```suggestion
target_include_directories(LDtkRaylib PRIVATE ../../include ${raylib_INCLUDE_DIRS})
target_link_libraries(LDtkRaylib PRIVATE ${raylib_LIBRARIES} LDtkLoader)
```
By the way, I was not able to successfully run without making this change.